pkg/jenkinsfile/pipeline.go

```diff
@@ -933,6 +933,10 @@ func (c *PipelineConfig) CreatePipelineForBuildPack(args CreatePipelineArguments
 	// If agent.container is specified, use that for default container configuration for step images.
 	containerName := c.Agent.GetImage()
 	if containerName != "" {
+		gcrRemoved := strings.TrimPrefix(containerName, "gcr.io/jenkinsxio/builder-")
```

Comment:
Is this related?

Reply:
See #6811 - the first commit in this PR is me finding a hackish way to actually test the PR image in the `charts` build pack. Just a hack, not to be merged - we need a real solution for #6811 down the road.